tests/dma/vif/viftestctx.cpp

```diff
@@ -0,0 +1,4 @@
+#include "viftestctx.h"
+
+TEST_CONTEXT testContext0 = { (u8*)0x11004000, VIF::VIF0, VIF::VIF0_FIFO, DMA::D0 };
+TEST_CONTEXT testContext1 = { (u8*)0x1100C000, VIF::VIF1, VIF::VIF1_FIFO, DMA::D1 };
```

Comment:
Rather than naming this a "test context", I'm thinking it may make more sense to put it in `VIF::`.  Still thinking about this more...

-[Unknown]